src/main/java/duke/application/TaskList.java

```diff
@@ -0,0 +1,61 @@
+package duke.application;
+import duke.task.Task;
+import java.util.ArrayList;
+
+/**
+ * TaskList - Represents a list of tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructs an empty TaskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Gets specific task.
+     */
+    public Task getTask(int i) {
+        return tasks.get(i);
+    }
+
+    /**
+     * Gets the list of tasks.
+     *
+     * @return ArrayList of tasks.
+     */
+    public ArrayList<Task> getTaskList() {
```

Comment:
Nice that you added a header comment for this method, but it is actually not required since it's a getter :) Noticed that you omitted the header comment for some others e.g. getDescription() from your Task class, can try to keep it consistent across the different classes as to whether you'd want to include or omit header comments for getters and setters!